resources/build/spritesvg.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * The external dependencies.
+ */
+const SvgSpriteLoaderPlugin = require('svg-sprite-loader/plugin');
+
+module.exports = new SvgSpriteLoaderPlugin({
+  plainSprite: true,
+  spriteAttrs: {
+    'aria-hidden': 'true',
+    style: 'position: absolute; width: 0; height: 0; overflow: hidden;',
```

Comment:
Is there any reason why we can't use `display: none; visibility: hidden;` here instead of absolute positioning?

Reply:
In really, no reason, is that because in icomoon give me this set of style, but was removed until this is not really necessary.